samples/samples/autocommit.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+import argparse
+
+from google.cloud.spanner_dbapi import connect
+
+# [START enable_autocommit_mode]
```

Comment:
Start tag should probably be moved to the start of the function itself, so that sample will not be function, just few lines of code.

Reply:
Done